src/index.js

```diff
@@ -0,0 +1,113 @@
+const MAX_TIMEOUT_ATTEMPTS = 3;
+const MAX_TIMEOUT = 1000;
+
+
+function main(client) {
+	let timeoutAttempts = 0;
+	let appliedClasses = [];
+	let unappliedFunctions = new Set();
+	let appliedFunctions = new Set();
+
+	function retrieveEvolvCssAsset() {
+		const styleSheets = document.getElementsByTagName("link");
+
+		for (let i = 0; i < styleSheets.length; i++) {
+			const link = styleSheets[i];
+			if (link.href &&
+					link.href.indexOf('evolv.ai') >= 0 &&
```

Comment:
I can foresee a scenario where assets.css is served from perhaps `localhost`. Perhaps we can use a different marker to identify Evolv-related stylesheets, such as a specific `data-` attribute?

Reply:
This can also be simplified down to the following by using attribute selectors.
```
return document.querySelector('link[href *= "evolv.ai"][href $= "assets.css"]');
```